exercises/practice/sieve/.meta/reference/sieve.bal

```diff
@@ -0,0 +1,43 @@
+
+# Returns the prime numbers less than or equal to the given limit.
+#
+# + 'limit - as an int
+# + return - prime numbers as an array of int
+public function primes(int 'limit) returns int[] {
+    boolean[] candidates = [];
+
+    candidates.push(false);
+    candidates.push(false);
+    foreach int _ in 2 ... 'limit {
+        candidates.push(true);
+    }
+
+    var markMultiples = function(int prime) {
+        int step = prime == 2 ? prime : prime * 2;
+        int i = prime * prime;
+        while i <= 'limit {
```

Comment:
As you've pointed out, there's an `int:range` function now. Shall we use the same here and instead of the while loop starting L28?

Reply:
I guess we can merge this PR with the current code and update it once the version update is done.